server/container/core/src/main/java/org/apache/james/core/MimeMessageCopyOnWriteProxy.java

```diff
@@ -67,24 +67,10 @@ public MessageReferenceTracker(MimeMessage ref) {
         }
 
         protected synchronized void incrementReferenceCount() {
-            /*
```

Comment:
I totally agree on that line removing when I read this class those comment confuse me more than anything. :+1: